### PR TITLE
ADAPT-10819 Fix: Missing tabIndex on flipcard items

### PR DESCRIPTION
**Pull request by ethan-lp**
fixes [ADAPT-10817](https://learningpool.atlassian.net/browse/ADAPT-10817)

### Fix
Ensured tabIndex on item to allow it to receive the class .focus-visible. Also updated 2 instances to uppercase 'Index' as error was flagged in console.

### CODE REVIEW

File: templates/flipcard.jsx [hunk 1]
```diff
@@ -19,11 +19,12 @@ export default function flipcard(props) {
               `component__item flipcard__item item-${index} ${_flipDirection}`,
             ])}
             key={index}
+            tabIndex='0'
```
Comment (cahirodoherty-learningpool) [inline templates/flipcard.jsx]: Is this one needed? I have a feeling it may disrupt the a11y flow of the component

Comment (ethan-lp) [reply templates/flipcard.jsx]: It is needed to allow the item to take the focus-visible class, however with or without it I am seeing the same behaviour / a11y flow using keyboard and enabling voiceover.

File: templates/flipcard.jsx [hunk 2]
```diff
@@ -19,11 +19,12 @@ export default function flipcard(props) {
...
           >
             <div
               className='flipcard__item-face flipcard__item-front'
               role='button'
-              tabindex='0'
+              tabIndex='0'
```
Comment (eleanor-heath) [inline templates/flipcard.jsx:26]: I dont think the I of index needs to be capitalised, please refer to this https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Review by eleanor-heath — APPROVED** [review]

File: templates/flipcard.jsx (hunk 2 above)
Comment (cahirodoherty-learningpool) [reply templates/flipcard.jsx:26]: Thats the mistake I made too Eleanor. HTML-wise you are correct, but React needs the camel case version - and then it changes this to the lowercase for output

**Comment by ethan-lp** [comment]
@cahirodoherty-learningpool @eleanor-heath I've removed the tab index I introduced but left in the capitalised ones, this does not solve the focus ring issue but that will be address in another ticket.

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**